lib/google/longrunning/operations_client.rb

```diff
@@ -124,7 +124,8 @@ def initialize \
           updater_proc = credentials.updater_proc
         end
 
-        defaults = default_settings(metadata, lib_name, lib_version)
+        _default_timeout = default_timeout
```

Comment:
nit: why not just call this something else rather than use `_`?

Reply:
Oh yeah, I have this different in other working branches. I'll rename this. I think `default_timeout` goes away, since it should use `timeout` here anyways.

Reply:
Update.